internal/framework/base/schema.go

```diff
@@ -84,6 +104,23 @@ func baseSchema() map[string]schema.Attribute {
 	}
 }
 
+func baseEphemeralSchema() map[string]ephemeralschema.Attribute {
+	return map[string]ephemeralschema.Attribute{
+		consts.FieldNamespace: ephemeralschema.StringAttribute{
+			Optional:            true,
+			MarkdownDescription: "Target namespace. (requires Enterprise)",
+			Validators: []validator.String{
+				validators.PathValidator(),
+			},
+		},
+		consts.FieldMountID: ephemeralschema.StringAttribute{
+			Optional: true,
```

Comment:
should this be required?

Reply:
Good question. This does not need to be `Required`. In the cases where a resource is ready to use and the ephemeral resource does not need to wait to provision, this field may be omitted, and the ephemeral resource can be used in the `Plan` stage

Reply:
hmm, seems like a good argument to remove this from the base schema for ephemeral types and have each ephemeral resource define and require what it needs.